cmd/core-command/Dockerfile

```diff
@@ -36,6 +36,9 @@ COPY . .
 
 RUN make cmd/core-command/core-command
 
+# Create simulated /etc/passwd file with a single user
+RUN echo "edgex_user:x:2002:2001:Linux User,,,:/home/edgex_user:/sbin/nologin" >> /tmp/passwd
+
```

Comment:
This will change when I pull in the Alpine changes to Scratch images from @jim-wang-intel - however, for future reference, this is how you force a Scratch based image to run as a specific user_id.

Reply:
I think this will land before Jim's PR, so IMO you should make the change from `scratch` to `alpine`